packages/dds/sequence/package.json

```diff
@@ -82,6 +83,7 @@
     "uuid": "^8.3.1"
   },
   "devDependencies": {
+    "@fluid-internal/attributor": ">=2.0.0-internal.2.3.0 <2.0.0-internal.3.0.0",
```

Comment:
i don't see this used? did i miss it? I'm concerned that this breaks our layering. even as a dev dependency i'm worried it could introduce diamond dependencies in our build that could cause build problems.

Reply:
talked to Tony offline, the DDS -> container-runtime implementation dep is what's problematic. This dependency is strictly to write some tests that target total size impact of framework attribution (both merge-tree attribution data and attributor contents). Plan is to either move those tests to this package or move them to functional-tests.